app/models/trip.rb

```diff
@@ -0,0 +1,6 @@
+class Trip < ApplicationRecord
+  belongs_to :driver
+  belongs_to :passenger
+  validates :date, presence: true
+  validates :cost, presence: true
```

Comment:
You should have a validation for rating to limit them between 0 and 5 or 1 and 5.